first-analysis-steps/split-jobs.md

```diff
@@ -28,6 +28,12 @@ and reduces the impact of jobs failing due to grid problems.
 Setting `filesPerJob=5` should work well for real data, while `filesPerJob=1` should be good for signal MC.
 {% endcallout %} 
 
+{% callout "Splitter arguments" %}
+The splitter has other useful arguments: 
+- `maxFiles`      : the maximal total number of files. By default splitter will run over all files in the dataset (corresponds to default value -1)
```

Comment:
* "by default _the_ splitter"
* "_which_ corresponds to _the_ default value _of_ -1"